packages/patrol/example/integration_test/permissions/notifications_test.dart

```diff
@@ -11,6 +11,16 @@ void main() {
         await $.native.grantPermissionWhenInUse();
       }
 
+      // Android 14+ requires additional permission to schedule notifications
+      try {
+        await $.native.tap(
+          Selector(text: 'Allow setting alarms and reminders'),
+        );
```

Comment:
Doesn't the default `grantPermission` work in this case? Because it should and if it doesn't it's very likely a bug (that warrants a new issue).

Reply:
It doesn't work, but it's not a dialog, it opens the settings page. Should `grantPermissionWhileInUse` also work in this case?
![Screenshot_1700218874](https://github.com/leancodepl/patrol/assets/37781044/a29de7e8-5034-457c-82f1-b0928c82d7c4)

Reply:
Ah, okay. It's okay to leave it as it is.